doc/api/api_changes.rst

```diff
@@ -59,6 +59,34 @@ which will improve the handling of log-axes.  Note that the
 returned *image* now is of type `~.matplotlib.collections.QuadMesh`
 instead of `~.matplotlib.image.AxesImage`.
 
+`.matplotlib.axes.Axes.get_tightbbox` now includes all artists
+--------------------------------------------------------------
+
+For Matplotlib 3.0, *all* artists are now included in the bounding box
+returned by `.matplotlib.axes.Axes.get_tightbbox`.
+
+`.matplotlib.axes.Axes.get_tightbbox` adds a new kwarg ``bbox_extra_artists``
+to specify the manually list of artists on the axes to include in the
```

Comment:
not a native speaker, but I feel the "manually" should either be an adjective rather than an adverb or be in another position in the sentence.

Reply:
Agreed!